src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Command runs certain commands input by user.
+ */
+public abstract class Command {
+    boolean isExit;
```

Comment:
```suggestion
    boolean isExitCommand;
```
Perhaps it might be better to use isExitCommand as a variable name instead of isExit, so that it is more intuitive when referring to this boolean in other classes?